integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -21,7 +21,7 @@ trap '${kubernetes_dir}/cleanup_env.sh' EXIT
 
 # Docker is required to initialize kubeadm, even if we are
 # using cri-o as the runtime.
-systemctl is-active --quiet docker || sudo systemctl start docker
+[ -n "${USE_PODMAN:-}" ] && (systemctl is-active --quiet docker || sudo systemctl start docker)
```

Comment:
I remember this was needed in the past, maybe it is not the case anymore. If you found on your testing that this is no longer required, we can get rid of this line.

Reply:
Removed.